CMakeLists.txt

```diff
@@ -72,7 +72,9 @@ file(GLOB_RECURSE HEADER_FILES
 file(GLOB_RECURSE FORM_FILES
         ${PROJECT_SOURCE_DIR}/code/gui/*.ui)
 
+add_python_target(code/scripts/__init__.py)
 add_python_target(code/scripts/robot.py)
+add_python_target(code/scripts/testScript.py)
```

Comment:
is there anyway to do this at runtime?

Reply:
Copying python scripts into the scripts folder? You can save scripts directly into it. This is for python code that is copied when building the project (like building c++ code)